frontend/src/entrypoints/archive_viewer.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020, Verizon Media
+// Licensed under the terms of the MIT. See LICENSE file in project root for terms.
+
+import { HashRouter } from 'react-router-dom'
+import { bootApp } from 'src/app_root'
+import { fetchJsonp } from 'src/helpers/fetch_jsonp'
+import { makeArchiveDataSource, OperationArchiveData } from 'src/services/data_sources/archive'
+
+fetchJsonp('archiveJsonp', 'data.js')
```

Comment:
This should be `data.json` I'll make a commit to address this, if you haven't already